### PR TITLE
iftop: add test

### PR DESCRIPTION
**Pull request by zmwangx**
#11898.

### CODE REVIEW

File: Formula/iftop.rb [hunk 1]
```diff
@@ -27,4 +27,8 @@ def caveats; <<-EOS.undent
     You should be certain that you trust any software you grant root privileges.
     EOS
   end
+
+  test do
+    assert_match "interface:", pipe_output("#{sbin}/iftop -t -s 1 2>&1")
```
Comment (ilovezfs) [inline Formula/iftop.rb]: what's the exit code? we should be able to use shell_output and assert it

Comment (zmwangx) [inline Formula/iftop.rb]: There are two possible exit codes. 1 if you use sandbox and it fails to read the device file. 0 if you --no-sandbox. Which is why I'm using pipe_output.

Comment (ilovezfs) [inline Formula/iftop.rb]: OK

**Comment by ilovezfs** [comment]
:tophat:

**Pull request merged**